tests/tests_lite/conftest.py

```diff
@@ -1,7 +1,104 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
For the future: We can add these in a last step. In fact, I think there is a pre-commit action that does it automatically